administrator/language/en-GB/en-GB.com_content.ini

```diff
@@ -126,6 +127,7 @@ COM_CONTENT_N_ITEMS_UNFEATURED="%s articles unfeatured."
 COM_CONTENT_N_ITEMS_UNFEATURED_1="%s article unfeatured."
 COM_CONTENT_N_ITEMS_UNPUBLISHED="%s articles unpublished."
 COM_CONTENT_N_ITEMS_UNPUBLISHED_1="%s article unpublished."
+COM_CONTENT_OTHER_OPTIONS="Options"
```

Comment:
This will need another name as you cant have a fieldset called options with a nested fieldset inside also called options

Reply:
Agree, I just took over the default, the naming need some fine adjustments.